core/sys/windows/user32.odin

```diff
@@ -17,6 +17,17 @@ foreign user32 {
 
 	GetClassNameW :: proc(hWnd: HWND, lpClassName: LPWSTR, nMaxCount: c_int) -> c_int ---
 
+	GetParent :: proc(hWnd: HWND) -> HWND ---
+	IsWindowVisible :: proc(hWnd: HWND) -> BOOL ---
+	SetWinEventHook :: proc(
+		eventMin, eventMax: DWORD,
+		hmodWinEventProc: HMODULE,
+		pfnWinEvenProc: WINEVENTPROC,
+		idProcess, idThread, dwmFlags: DWORD,
```

Comment:
I'd add the enum for `dwmFlags`, or even make it a `bit_set`.
```cpp
        WINEVENT_INCONTEXT = 4 // 2 in bit_set
        WINEVENT_OUTOFCONTEXT = 0  // not present in a bit_set
        WINEVENT_SKIPOWNPROCESS = 2 // 1 in bit_set
        WINEVENT_SKIPOWNTHREAD = 1 // 0 in bit_set
```
Keep in mind that for the `bit_set`, passing `WINEVENT_OUTOFCONTEXT` would mean an empty bit set `{}`

Reply:
On a related note that's not your fault, I find it a bit sloppy that [Microsoft](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwineventhook) uses `DWORD` and `UINT` interchangeably, using `DWORD` in the signature, but then mixing `UINT` and `DWORD` under `Parameters`. It's ultimately the same type, so whatever, but sloppy documentation work.